src/xrCDB/xr_area.cpp

```diff
@@ -106,33 +106,34 @@ void CObjectSpace::Load(IReader* F, CDB::build_callback build_callback)
     F->r(&H, sizeof(hdrCFORM));
     Fvector* verts = (Fvector*)F->pointer();
     CDB::TRI* tris = (CDB::TRI*)(verts + H.vertcount);
+    Static.set_version(F->get_age());
     Create(verts, tris, H, build_callback);
     FS.r_close(F);
 }
 
-void CObjectSpace::Create(Fvector* verts, CDB::TRI* tris, const hdrCFORM& H, CDB::build_callback build_callback)
+void CObjectSpace::Create(Fvector* verts, CDB::TRI* tris, const hdrCFORM& H,
+                          CDB::build_callback build_callback)
```

Comment:
No unrelated formatting changes needed.
```suggestion
void CObjectSpace::Create(Fvector* verts, CDB::TRI* tris, const hdrCFORM& H, CDB::build_callback build_callback)
```